specification/onlineexperimentation/Azure.Analytics.OnlineExperimentation/tspconfig.yaml

```
@@ -0,0 +1,56 @@
+parameters:
+  service-dir:
+    default: "sdk/onlineexperimentation"
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/data-plane"
+options:
+  "@azure-tools/typespec-autorest":
+    output-file: "{azure-resource-provider-folder}/Microsoft.OnlineExperimentation/{version-status}/{version}/onlineexperimentation.json"
+    emitter-output-dir: "{project-root}/../"
+    azure-resource-provider-folder: "data-plane"
+    examples-dir: "{project-root}/examples"
+    omit-unreachable-types: true
+    emit-lro-options: "none"
+  "@typespec/openapi3":
+    emitter-output-dir: "{project-root}/../"
+    output-file: "{output-folder-base}/data-plane/OpenApi3/Microsoft.OnlineExperimentation/preview/{version}/onlineexperimentation.json"
+  "@azure-tools/typespec-go":
+    module: "github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}"
+    service-dir: "sdk/analytics"
+    package-dir: "azonlineexperimentation"
+    module-version: "0.0.1"
+    generate-fakes: true
+    inject-spans: true
+    single-client: true
+    slice-elements-byval: true
+  "@azure-tools/typespec-python":
+    package-dir: "azure-analytics-onlineexperimentation"
+    package-name: "{package-dir}"
```

Per python architect feedback:
```suggestion
    package-dir: "azure-onlineexperimentation"
    package-name: "{package-dir}"
    namespace: "azure.onlineexperimentation"
```